org.libvips.nip4.json

```diff
@@ -0,0 +1,361 @@
+{
+    "app-id" : "org.libvips.nip4",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "nip4",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=host"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/libheif",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/lib/openjpeg-*",
+        "/lib/*.a",
+        "/man",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/gir-1.0",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/aclocal*",
+        "/share/autoconf*",
+        "/share/automake*",
+        "/share/info",
+        "/share/thumbnailers",
+        "/share/USING_HDF5_CMake.txt",
+        "/share/COPYING",
+        "/share/RELEASE.txt",
+        "/share/cmake",
+        "/bin/smem",
+        "/bin/fpack",
+        "/bin/funpack",
+        "/bin/speed",
+        "/bin/vipsedit",
+        "/bin/vipsprofile",
+        "/bin/vipsthumbnail"
+    ],
+    "modules" : [
+
+        {
+            "name" : "highway",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "build-options" : {
+                "config-opts" : [
+                    "-DBUILD_TESTING=OFF",
+                    "-DHWY_ENABLE_CONTRIB=OFF",
+                    "-DHWY_ENABLE_EXAMPLES=OFF",
+                    "-DHWY_ENABLE_TESTS=OFF"
+                ]
+            },
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/google/highway.git",
+                    "tag" : "1.2.0",
+                    "commit" : "457c891775a7397bdb0376bb1031e6e027af1c48"
+                }
+            ]
+        },
+
+        {
+            "name" : "libdeflate",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/ebiggers/libdeflate.git",
+                    "tag" : "v1.22",
+                    "commit" : "2335c047e91cac6fd04cb0fd2769380395149f15"
+                }
+            ],
+	    "cleanup" : [
+		"/bin/libdeflate-gunzip",
+		"/bin/libdeflate-gzip"
```

Comment:
shouldn't this just be `/bin`

Reply:
that's how `cleanup` works.

Reply:
I assumed I had to name them explicitly. I'll change this, thanks.

Reply:
from the man

>        cleanup (array of strings)
>           An array of file patterns that should be removed at the end. Patterns starting with / are taken to be full pathnames
>           (without the /app prefix), otherwise they just match the basename. Note that any patterns will only match files
>           installed by this module.